include/ismrmrd/ismrmrd.h

```diff
@@ -132,89 +131,88 @@ EXPORTISMRMRD size_t ismrmrd_sizeof_data_type(int data_type);
  * Acquisition Flags
  */
 enum ISMRMRD_AcquisitionFlags {
-    ISMRMRD_ACQ_FIRST_IN_ENCODE_STEP1               =  1,
```

Comment:
Was all the whitespace removal intentional?

Reply:
Nope, that as was a mistake. Switching off automatic formatting for these files.